lua/entities/gmod_tardis/modules/sh_thirdperson.lua

```diff
@@ -13,6 +13,17 @@ TARDIS:AddKeyBind("tp-toggledoor",{
 	exterior=true
 })
 
+TARDIS:AddSetting({
+	id="thirdperson_careful_enabled",
+	name="Use walk key to open third person",
+	desc="Should the WALK ('ALT' by default) key be pressed to enter third person by clicking the console?",
```

Comment:
Recommend minor language improvement here, not sure how this will look ingame though

```suggestion
	desc="Should the WALK ('ALT' by default) key be pressed to enter third person when pressing USE ('E' by default) key on the console?",
```